### ISSUE TITLE
4 images in a row on pacific epping

### ISSUE THREAD

**Issue by LaraStudioFriday** [markendley/plmpndspry#49]
How do i get this page - http://marke2.sg-host.com/plumpspry/case-study/pacificepping/

to look like this with 4 in a row though?

<img width="1244" alt="Screen Shot 2019-12-11 at 2 19 34 pm" src="https://user-images.githubusercontent.com/55725505/70588576-4d06a100-1c21-11ea-94d6-68ad315195c6.png">

**Comment by markendley** [comment]
Choose "Slider (3 at desktop)" from the radio buttons

**Comment by LaraStudioFriday** [comment]
Why don't they have the same ratio? The girl is cropped out too - can we show more of the images?

<img width="1275" alt="Screen Shot 2019-12-12 at 10 24 17 am" src="https://user-images.githubusercontent.com/55725505/70669154-a7f3d300-1cc9-11ea-8b7c-d18ab79ab619.png">
<img width="1269" alt="Screen Shot 2019-12-12 at 10 24 01 am" src="https://user-images.githubusercontent.com/55725505/70669159-ac1ff080-1cc9-11ea-844f-1e9a1169c45f.png">

Also, the phone looks weird - Paul liked it the way it was before:

![79155730_2532379633477786_7953624853406285824_n](https://user-images.githubusercontent.com/55725505/70669322-22245780-1cca-11ea-8afc-528b23a734a3.png)
<img width="445" alt="Screen Shot 2019-12-12 at 10 28 54 am" src="https://user-images.githubusercontent.com/55725505/70669351-3ec08f80-1cca-11ea-9607-0375495a3b5e.png">

**Comment by markendley** [comment]
Well to answer your question, it's because 3 per row needed a different ratio than 4 per row.

I created you another slider option but then looked at the images you've uploaded for that slider.
http://marke2.sg-host.com/plumpspry/case-study/pacificepping/

Then I got confused...
You've put landscape images in the Desktop
and portrait images on mobile.
That doesn't work for that slider that I created for Connoh

So I did this: http://marke2.sg-host.com/plumpspry/case-study/pacificepping/
